### DISCUSSION TITLE
Is there a way to add folder?

### DISCUSSION THREAD

**Discussion by tombohub** [ionide/ionide-vscode-fsharp#1878] [Q&A]
Is there a way to add folder you we can add files? I don't see the option:

![image](https://github.com/ionide/ionide-vscode-fsharp/assets/3029412/cd6c3d6f-0581-4e92-89f4-32451706c36a)

**Comment by MangelMaxime** [comment]
Currently, you can't add an empty folder.

However, you can specify a path like `Types/file.fs` and the `Types` folder will be created if not already existing.
